pkg/readiness/readiness.go

```diff
@@ -60,30 +59,48 @@ func Run(bird, bird6, felix bool) {
 // checkBIRDReady checks if BIRD is ready by connecting to the BIRD
 // socket to gather all BGP peer connection status, and overall graceful
 // restart status.
-func checkBIRDReady(ipv string) error {
+func checkBIRDReady(ipv string, thresholdTime time.Duration) error {
 	// Check for unestablished peers
 	peers, err := bird.GetPeers(ipv)
 	log.Debugf("peers: %v", peers)
 	if err != nil {
 		return err
 	}
 
+	// numEstablishedPeer keeps count of number of peers with bgp state established
+	numEstablishedPeer := 0
 	s := []string{}
+
 	for _, peer := range peers {
-		if peer.BGPState != "Established" {
+		if peer.BGPState == "Established" {
+			numEstablishedPeer += 1
+		} else if peer.BGPState != "Established" {
```

Comment:
The `if peer.BGPState != "Established"` is redundant here since the only way to reach the `else` block is if `peer.BGPState != "Established"`